### PR TITLE
feature: Add Identity System API Endpoint

### PR DESCRIPTION
**Pull request by dhedey**
* Exposes the node's public key and various other identifiers
* Configures envsubst for docker

### CODE REVIEW

File: core/src/main/java/com/radixdlt/RadixNodeModule.java [hunk 1]
```diff
@@ -205,18 +206,19 @@ protected void configure() {
     // Consensus
     final String useGenesis = properties.get("consensus.use_genesis_for_validator_address");
     final String validatorAddress = properties.get("consensus.validator_address", (String) null);
-    if (useGenesis != null && validatorAddress != null) {
+    if (!Strings.isNullOrEmpty(useGenesis) && !Strings.isNullOrEmpty(validatorAddress)) {
```
[the remark this answers is not in the document]
Comment (dhedey) [reply core/src/main/java/com/radixdlt/RadixNodeModule.java:209]: ohhh write yes this logic is wrong! Good point Will fix!

File: core/src/main/java/com/radixdlt/api/system/routes/IdentityHandler.java [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+import com.radixdlt.crypto.ECDSASecp256k1PublicKey;
+import com.radixdlt.p2p.NodeId;
+import com.radixdlt.p2p.RadixNodeUri;
+import java.util.Optional;
+
+public final class IdentityHandler extends SystemGetJsonHandler<IdentityResponse> {
+
+  private final Addressing addressing;
+
+  @SuppressWarnings("OptionalUsedAsFieldOrParameterType")
```
Comment (LukasGasior1) [inline core/src/main/java/com/radixdlt/api/system/routes/IdentityHandler.java:83]: java: adds Optional type
devs: use Optional
java: \<surprised pikachu face\>

**Review by LukasGasior1 — APPROVED** [review]
LGTM :+1:

File: core/src/main/java/com/radixdlt/api/system/routes/IdentityHandler.java (hunk 2 above)
Comment (dhedey) [reply core/src/main/java/com/radixdlt/api/system/routes/IdentityHandler.java:83]: It's literally how OptionalBinder works 🤦 http://google.github.io/guice/api-docs/latest/javadoc/com/google/inject/multibindings/OptionalBinder.html

**Pull request merged**